client/pages/about-us/profile-Card.js

```diff
@@ -0,0 +1,34 @@
+import ProfileComponent from "./profileCard-component.js";
+
+const ProfileCard = {
+  components: {
+    ProfileComponent,
+  },
+  template: `
+  <div class="About-main">
+  <div class="profile-card">
+    <template v-for="profile in profiles">
+      <profile-component:profile="profile"/>
+    </template>
+  </div>
+  </div>
+  `,
+  data() {
+    return {
+      Profiles: [
+        { img: "/images/members/MoaminA.png", caption: "Moamine","Full stack developer"},
```

Comment:
There is an error in the json.
It should be something like:

`{ img: "/images/members/MoaminA.png", title: "Moamine", subtitle:"Full stack developer"},`